package.json

```diff
@@ -34,18 +34,23 @@
 	"scripts": {
 		"build": "yarn clean && lerna run build --stream",
 		"clean": "lerna run clean",
-		"ganache": "npx ganache -m 'smart guide what forget tired jungle always expire rescue boring glue champion' --account=\"0x4c3758228f536f7a210f8936182fb5b728046970b8e3215d0b5cb4c4faae8a4e,100000000000000000000\" --account=\"0x34aeb1f338c17e6b440c189655c89fcef148893a24a7f15c0cb666d9cf5eacb3,100000000000000000000\"",
-		"ganache-background": "yarn ganache > /dev/null & client=$! npx wait-port 8545",
-		"geth:build": "docker build ./.github -t gethnode -f .github/Dockerfile",
-		"geth": "docker run -p 8545:8545 gethnode --mine --miner.noverify --datadir data --networkid 4224 --nodiscover --nat \"any\" --ws --ws.addr 0.0.0.0 --ws.port 8545 --http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api personal,web3,eth,admin,debug,miner,txpool,net --ws.api personal,web3,eth,admin,debug,miner,txpool,net --unlock 0 --password /pass",
-		"geth-background": "docker run -d -p 8545:8545 gethnode --mine --miner.noverify --datadir data --networkid 4224 --nodiscover --nat \"any\" --ws --ws.addr 0.0.0.0 --ws.port 8545 --http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api personal,web3,eth,admin,debug,miner,txpool,net --ws.api personal,web3,eth,admin,debug,miner,txpool,net --unlock 0 --password /pass",
+		"ganache:start": "/bin/bash -c 'scripts/ganache.sh start'",
+		"ganache:stop": "./scripts/ganache.sh stop",
+		"geth:start": "/bin/bash -c 'scripts/geth.sh start'",
+		"geth:stop": "./scripts/geth.sh stop",
```

Comment:
Regarding the use of `/bin/bash -c` [this](https://stackoverflow.com/a/16365367) is just something to be aware of, looks like we don't have much of an options since we're passing the `start` and `stop` commands

Reply:
It's because we are setting up ENV variables inside the script that requires to run it in a bash shell so change can be passed to next script.